apps/meteor/client/views/setupWizard/providers/SetupWizardProvider.tsx

```diff
@@ -122,34 +121,34 @@ const SetupWizardProvider = ({ children }: { children: ReactElement }): ReactEle
 		]);
 	}, [dispatchSettings, setupWizardData]);
 
-	const saveOrganizationData = useCallback(async (): Promise<void> => {
-		const {
-			organizationData: { organizationName, organizationIndustry, organizationSize, country },
-		} = setupWizardData;
-
-		await dispatchSettings([
-			{
-				_id: 'Country',
-				value: country,
-			},
-			{
-				_id: 'Industry',
-				value: organizationIndustry,
-			},
-			{
-				_id: 'Size',
-				value: organizationSize,
-			},
-			{
-				_id: 'Organization_Name',
-				value: organizationName,
-			},
-		]);
-	}, [dispatchSettings, setupWizardData]);
+	const saveOrganizationData = useCallback(
+		async (organizationData: ContextType<typeof SetupWizardContext>['setupWizardData']['organizationData']): Promise<void> => {
+			const { organizationName, organizationIndustry, organizationSize, country } = organizationData;
+
+			await dispatchSettings([
+				{
+					_id: 'Country',
+					value: country,
+				},
+				{
+					_id: 'Industry',
+					value: organizationIndustry,
+				},
+				{
+					_id: 'Size',
+					value: organizationSize,
+				},
+				{
+					_id: 'Organization_Name',
+					value: organizationName,
+				},
+			]);
+		},
+		[dispatchSettings],
+	);
 
 	const registerServer: HandleRegisterServer = useMutableCallback(async ({ email, resend = false }): Promise<void> => {
```

Comment:
I suggest to remove this console.log from the `catch` block and add a toast message to show the user that has an error with the attempt

Reply:
ok!